src/components/subscriptions/LicenseActions.jsx

```diff
@@ -33,19 +53,19 @@ export default function LicenseAction({ user }) {
   return (
     <div className="license-actions">
       {licenseActions.map((licenseAction) => {
-        if (licenseAction.handleClick) {
-          return (
-            <button
-              key={licenseAction.text}
-              className="btn btn-link btn-sm p-0"
-              onClick={() => licenseAction.handleClick()}
-            >
-              {licenseAction.text}
-            </button>
-          );
-        }
-        return licenseAction.text;
-      })}
+          if (licenseAction.handleClick) {
```

Comment:
indentation of this code block is off, I think the convention in this project is to use 2 space indentation.